<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PieDataEditor</class>
 <widget class="QWidget" name="PieDataEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>192</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="3">
    <widget class="KoChart::ChartTableView" name="tableView"/>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>97</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="insertRow">
     <property name="text">
      <string>Insert</string>
     </property>
     <property name="icon">
      <iconset theme="edit-table-insert-row-under"/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="deleteSelection">
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="icon">
      <iconset theme="edit-table-delete-row"/>
     </property>
     <property name="shortcut">
      <string>Del</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KoChart::ChartTableView</class>
   <extends>QTableView</extends>
   <header>ChartTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
